tools/system_libs.py

```diff
@@ -744,14 +744,17 @@ def get_files(self):
     ignore += LIBC_SOCKETS
 
     # individual math files
-    ignore += [
-        'abs.c', 'cos.c', 'cosf.c', 'cosl.c', 'sin.c', 'sinf.c', 'sinl.c',
+    ignore += ['abs.c', 'round.c', 'roundf.c']
+    if not shared.Settings.WASM:
+      # if building to wasm, we need more math code, since we have fewer builtins
```

Comment:
the comment here might be clearer as "when not building to wasm" (since that's the if we are in).

btw I think the history here is that for asm.js we use some Math.* stuff from JS. And I guess that carried over to non-wasm in upstream. But we should probably remove the difference between wasm and non-wasm here, after fastcomp is gone.